### ISSUE TITLE
Update initial focus, when Profile subpage loads

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#108206]
A finding from our accessibility audit: https://github.com/department-of-veterans-affairs/va.gov-team/issues/107960#issuecomment-2822322212

URL: [staging.va.gov/profile](http://staging.va.gov/profile)
Expected severity level: 1

### Actual (Issue):
- When a user selects a link on the Profile hub and the subpage loads, sometimes focus moves to the subpage H1, and sometimes it moves to the “Profile” H2 for the lefthand menu. [Demonstration (Zoom clip)](https://coformaco.zoom.us/clips/share/-Cc1yz84THyjWn11cAlDUQ)
- “Manage your contact information,” “Check your accredited representative or VSO,” and “Review account security” send focus to the “Profile” H2
- All other links send focus to the H1 (“Personal information” on the [Personal information page](https://staging.va.gov/profile/personal-information), for example)
- This should be consistent across the app; otherwise screen reader users may get lost and have trouble navigating on those 3 pages where focus isn’t managed properly.

### Expected: 
- When the subpage loads, initial focus should be on the “Accredited Representative of VSO” H1. 
- Focus should NOT be on the "Profile" H2 in the lefthand menu. 

----
**We can offer to help fix these two pages too:**
“Manage your contact information”
“Review account security”

**Comment by mtri** [comment]
I tested with each type of representative test user (VSO, attorney, claims agent) and this appears to be working correctly.
